langchain/vectorstores/chroma.py

```diff
@@ -574,6 +574,51 @@ def from_documents(
             **kwargs,
         )
 
+    @classmethod
+    def from_embeddings(
+        cls: Type[Chroma],
+        text_embeddings: List[Tuple[str, List[float]]],
+        embedding: Optional[Embeddings] = None,
+        metadatas: Optional[List[dict]] = None,
+        ids: Optional[List[str]] = None,
+        collection_name: str = _LANGCHAIN_DEFAULT_COLLECTION_NAME,
+        persist_directory: Optional[str] = None,
+        client_settings: Optional[Any] = None,
+        client: Optional[Any] = None,
+        **kwargs: Any,
+    ) -> Chroma:
+        """Create a Chroma vectorstore from a list of embeddings.
+
+        If a persist_directory is specified, the collection will be persisted there.
+        Otherwise, the data will be ephemeral in-memory.
+
+        Args:
+            text_embeddings (List[Tuple[str, List[float]]]): List of tuples with texts and embeddings (list of floats).
+            metadatas (Optional[List[dict]]): List of metadatas. Defaults to None.
+            collection_name (str): Name of the collection to create.
+            persist_directory (Optional[str]): Directory to persist the collection.
+            ids (Optional[List[str]]): List of document IDs. Defaults to None.
+            embedding (Optional[Embeddings]): Embedding function. Defaults to None.
+            client_settings (Optional[chromadb.config.Settings]): Chroma client settings
+
+        Returns:
+            Chroma: Chroma vectorstore.
+        """
+        chroma_collection = cls(
+            collection_name=collection_name,
+            embedding_function=embedding,
+            persist_directory=persist_directory,
+            client_settings=client_settings,
+            client=client,
+        )
+        # split the text_embeddings into text and embeddings
+        texts = [item[0] for item in text_embeddings]
+        embeddings = [item[1] for item in text_embeddings]
+        chroma_collection.add_texts(
```

Comment:
i don't think add_texts actually takes into account embeddings that are passed in? maybe something worth adding

Reply:
Indeed, thanks for pointing this out @baskaryan! This was actually a copy-paste error, I did add the new parameter in my local version but overlooked it when transfering it to my langchain fork. I updated the PR, let me know what you think!